### ISSUE TITLE
A couple bugs: Beer, UI size, and main menu

### ISSUE THREAD

**Issue by Animatical** [MSCLoaderPro/MSCModLoaderPro#5]
**Describe the bug**
I'll start with the beer issu- EDIT: Just found out it was MOP. 

Next, I've noticed that the UI size itself shrunk when playing on a resolution lower than 1280x720. Again, small bug.

Lastly, I've noticed that after saving and exiting the game, the music in the main menu disappears, even though I didn't turn it off.

**To Reproduce**
For the UI issue, there are no steps other than just entering the game at, for example, 800x600 or 1024x768.

For the main menu music issue, simply:
1. Open my summer car
2. Enter any save game
3. Do anything, doesn't matter
4. Save and exit the game
5. The music will be gone.

**Expected behavior**
For the UI, It shouldn't be a different size than the UI at 1920x1080
And for the main menu, it should still play music even after saving.

**output_log.txt**
[output_log.txt](https://github.com/MSCLoaderPro/MSCModLoaderPro/files/6430247/output_log.txt)

**Additional context**
I've only noticed these bugs after installing the Mod Loader Pro.
Here's a mod list, if you need it.

Mods:
Attachable Roofrack
CarryMore
Dirtmenag (dirtiness disable)
Garage Pit Covers
Grab Anything
Mokia Townsman
MOP (pro version)
MscNoclip
Shed
Toiveradio Enhanced
Ventti bet simplification
WoodGrainDashboard

**Comment by Fredrik95** [comment]
Okay, we'll take a look, regarding the UI issue, I can't reproduce, it looks the same to me at lower resolutions.

Can you attach a few screenshot demonstrating the behavior?

**Comment by Animatical** [comment]
Wow, I went to try again and noticed it was fixed. The only mod missing from my folder is ventti bet simplification, which I don't have a clue how that could have caused it. Never had it with the previous mod loader, so maybe it's just another case of "old mod not so compatible with new loader"
Anyway, 2 problems have already been solved, but I know for a fact the menu music is a loader issue.
Thanks for your time :)
